README.md

```diff
@@ -23,6 +23,7 @@
 [![de](https://img.shields.io/badge/lang-de-black)](README.de.md)
 [![ita](https://img.shields.io/badge/lang-it-red)](README.it.md)
 [![cs](https://img.shields.io/badge/lang-cs-purple)](README.cs.md)
+[![cs](https://img.shields.io/badge/lang-nb-blue)](README.nb.md)
```

Comment:
```suggestion
[![nb](https://img.shields.io/badge/lang-nb-blue)](README.nb.md)
```

Just set the correct code for your language in all README files, please

Reply:
@alexhostrup, could you just adjust these?

Reply:
I have fixed this now